devtools/conda-envs/docs.yaml

```
@@ -19,13 +19,14 @@ dependencies:
   - sphinxcontrib-bibtex
   - sphinx-notfound-page
   - myst-parser
+  - autodoc_pydantic
```

```suggestion
  - autodoc-pydantic
```

I think the spelling is different on conda! https://anaconda.org/conda-forge/autodoc-pydantic

---

Thanks Lily